Add candlestick tutorial (#916)
timescale/docs: timescaledb/tutorials/financial-candlestick-tick-data/design-tick-schema.md

**crypto_assets:**

| Field | Description |
|---|---|---|---|
| symbol | The symbol of the crypto currency pair, eg.: BTC/USD |
| name | The name of the pair, eg.: Bitcoin USD |

**crypto_ticks:**

| Field | Description |
|---|---|
| time | Timestamp of the price record in UTC time zone |
| symbol | Crypto pair symbol from the `crypto_assets` table |
| price | The price registered on the exchange |
| day_volume | Total volume for the given day (incremental) |

**Create the tables:**
```sql
CREATE TABLE crypto_assets (
    symbol UNIQUE TEXT,
    "name" TEXT
);
 
CREATE TABLE crypto_ticks (
    "time" TIMESTAMPTZ,
    symbol TEXT,
    price DOUBLE PRECISION,
    day_volume NUMERIC
);
 
SELECT create_hypertable('crypto_ticks', 'time');
```

Notice how you also need to turn the *time-series* table into a
[hypertable][hypertable]. This is an important step in order to efficiently
store your time-series data in TimescaleDB.

### Should I use TIMESTAMP or TIMESTAMP WITH TIME ZONE data type for the time column?
Our recommendation is to always use `TIMESTAMP WITH TIME ZONE` (`TIMESTAMPTZ`)
as the data type to store time values. This way the timestamp column holds the

> time zone information as well which will make it easier to query your data

Reviewer: ```suggestion
time zone information as well which makes it easier to query your data
```

using different time zones if needed. TimescaleDB has support for time zones
and it stores `TIMESTAMPTZ` values in UTC internally and makes the necessary

> conversions to satisfy your queries.

Reviewer: ```suggestion
conversions for your queries.
```

|day_volume|Total volume for the given day (incremental)|

Create the tables:
```sql
CREATE TABLE crypto_assets (
    symbol UNIQUE TEXT,
    "name" TEXT
);
 
CREATE TABLE crypto_ticks (
    "time" TIMESTAMPTZ,
    symbol TEXT,
    price DOUBLE PRECISION,
    day_volume NUMERIC
);
```

You also need to turn the time-series table into a [hypertable][hypertable]:
```sql
-- convert the regular 'crypto_ticks' table into a TimescaleDB hypertable with 7-day chunks
SELECT create_hypertable('crypto_ticks', 'time');
```

This is an important step in order to efficiently store your time-series
data in TimescaleDB.

### Using TIMESTAMP data types
It is best practice to store time values using the `TIMESTAMP WITH TIME ZONE` (`TIMESTAMPTZ`)
data type. This makes it easier to query your data
using different time zones. TimescaleDB 
stores `TIMESTAMPTZ` values in UTC internally and makes the necessary
conversions for your queries.

## Insert tick data
With the hypertable and relational table created, download the sample files
containing crypto assets and tick data from the last three weeks. Insert the data
into your TimescaleDB instance.

<procedure>

### Inserting sample data

> 1. Download [the sample `.csv` files][sample-download].

Reviewer 2: ```suggestion
1. Download the sample `.csv` files: <tag type="download">[crypto_sample.csv](https://assets.timescale.com/docs/downloads/crypto_sample.zip)
```

I'm actually not 100% sure if this will render correctly, but we should try to be consistent with the visual tag when downloading files IMO.

    ```bash
    wget https://assets.timescale.com/docs/downloads/crypto_sample.zip
    ```
1. Unzip the file.
    ```bash

>     unzip crypto_sample.zip

Reviewer 2: ```suggestion
    unzip crypto_sample.zip
    cd crypto_sample
```
Either this, or you'll need to change the text/example below to make sure it pulls from the correct folder.
